ckanext/og_datatablesview/plugin/flask_plugin.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from flask import Blueprint
+
+import ckan.plugins as p
+import ckanext.og_datatablesview.utils as utils
+
+
+ogdatatablesview = Blueprint(u'ogdatatablesview', __name__)
+
+
+class MixinPlugin(p.SingletonPlugin):
+    p.implements(p.IBlueprint)
+
+    # IBlueprint
+    def get_blueprint(self):
+        return ogdatatablesview
+
+
+def ajax(resource_view_id):
+    return utils.ajax(resource_view_id)
+
+def filtered_download(resource_view_id):
```

Comment:
it is ok to put utils.filtered_download to ogdatatablesview.add_url_rule instead of filtered_download and remove this stub functions.

Reply:
I believe so. The purpose of the stub function is because some of the code in `utils.filtered_download` and `utils.ajax` is dependent on the CKAN version. These functions are also used in `ckanext/og_datatablesview/plugin/pylons_plugin.py`

I used ckanext-dcat as an example:
- https://github.com/ckan/ckanext-dcat/blob/master/ckanext/dcat/utils.py
- https://github.com/ckan/ckanext-dcat/blob/master/ckanext/dcat/blueprints.py
- https://github.com/ckan/ckanext-dcat/blob/master/ckanext/dcat/controllers.py